fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveRemoteFileIO.java

```diff
@@ -188,7 +188,7 @@ private boolean isValidDirectory(FileStatus fileStatus) {
             return false;
         }
         String dirName = fileStatus.getPath().getName();
-        return !(dirName.startsWith("."));
+        return !(dirName.startsWith(".") || dirName.startsWith("_temporary"));
```

Comment:
ignore files that satisfies `dirName.startsWith("_")`, aligning with `isValidDataFile`

Reply:
get